GFS_layer/GFS_typedefs.F90

```diff
@@ -1196,6 +1197,11 @@ module GFS_typedefs
     real (kind=kind_phys), pointer :: gflux  (:)    => null()   !< groud conductive heat flux
     real (kind=kind_phys), pointer :: dlwsfc (:)    => null()   !< time accumulated sfc dn lw flux ( w/m**2 )
     real (kind=kind_phys), pointer :: ulwsfc (:)    => null()   !< time accumulated sfc up lw flux ( w/m**2 )
+    real (kind=kind_phys), pointer :: dswsfc (:)     => null()   !< time accumulated sfc dn sw flux ( w/m**2 ) when gfs_physics_nml.override_surface_radiative_fluxes == .true.
```

Comment:
These new `dswsfc`/`uswsfc` variables: are they intended to replace the `fluxr` array so they hold the true (overridden) surface fluxes? Was this because modifying `fluxr` could have undesirable side effects?